src/context/Data/DataProvider.tsx

```diff
@@ -626,8 +625,15 @@ export default class DataProvider extends React.Component<DataProviderProps, Dat
                         messageCid: messageID
                     }
                     callMetricsApi(requestNumber, EVENT_TYPE.DC_ALLOCATION, params, config.metrics_api_environment)
+                    return
                 }
 
+                await this.props.github.githubOcto.issues.addLabels({
```

Comment:
That should be in an else, right? In othercase will be executed always even if the first if changes the label to granted